src/adaptermanager.js

```diff
@@ -198,9 +210,17 @@ exports.makeBidRequests = function(adUnits, auctionStart, auctionId, cbTimeout,
 };
 
 exports.callBids = (adUnits, bidRequests, addBidResponse, doneCb) => {
-  let serverBidRequests = bidRequests.filter(bidRequest => {
-    return bidRequest.src && bidRequest.src === CONSTANTS.S2S.SRC;
-  });
+  if (!bidRequests.length) {
+    utils.logWarn('callBids executed with no bidRequests.  Were they filtered by labels or sizing?');
+    return;
+  }
+
+  let ajax = ajaxBuilder(bidRequests[0].timeout);
+
+  let [clientBidRequests, serverBidRequests] = bidRequests.reduce((partitions, bidRequest) => {
+    partitions[Number(typeof bidRequest.src !== 'undefined' && bidRequest.src === CONSTANTS.S2S.SRC)].push(bidRequest);
+    return partitions;
+  }, [[], []]);
 
   if (serverBidRequests.length) {
     let adaptersServerSide = _s2sConfig.bidders;
```

Comment:
`_s2sConfig` is not having any default values here. I am getting below object
```
{
  "enabled": true,
  "bidders": [
    "appnexusAst"
  ],
  "accountId": "<some-id>"
}
```
I tested with this [test page](https://jsfiddle.net/oy3mzh4p/)

Reply:
@snapwich 
Did you check this? Jaimin said it was broken for him when he tested.

Reply:
just tested with the latest and I seem to be getting the defaults.  I'll push the few changes I have and see if that helps, but I think it should have been working before as well.